int16 IntersectionID
int16 PhaseID
int16 RedTime
int16 GreenTime
int16 YellowTime